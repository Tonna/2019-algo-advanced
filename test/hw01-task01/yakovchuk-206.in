30
1 1 1 1 1 1 1 2 5 4 8 2 10 8 3 4 7 8 13 9 15 13 3 21 21 17 9 2 25
6
12 6
22 2
24 4
17 8
7 2
1 8
