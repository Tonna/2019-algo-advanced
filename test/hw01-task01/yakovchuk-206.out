13
23
6
11
8
8
